docs: Added managed flow required configuration (#586)
mmazzarolo/react-native-modal-datetime-picker: README.md

expo install react-native-modal-datetime-picker @react-native-community/datetimepicker
```

> You will also need to configure supported appearance styles in your app.json:

Reviewer: ```suggestion
You can configure appearance styles in your `app.json`:
```

Reviewer: It's not a requirement... right?

Reviewer: Also, I don't think the iOS interface can be customized until #549 lands, right?

Author: So I just implemented "react-native-modal-datetime-picker" in my project, it seems to be a requirement, if that config is not set up then the system color schema always returns "light".

Reviewer: Gotcha, I updated the suggestion :)

Author: Yeah, that's correct, if the "userInterfaceStyle": "automatic" config is not set up and the user has their device in dark mode what happens is that it looks like the modal is empty although it is actually not, it's white text on white background, the config lets your code understand what colour scheme the system is using. This config is not required for the modal to work, but I spent a few hours looking for why I wasn't able to see it and the config was the solution so I thought I would add it to the documentation so others didn't have to do the same.

Reviewer: Could you remove the `android` and `ios` bits so that we can merge it (as they're not needed)?

Author: Changes were pushed
